tests/gracenote_tests.js

```diff
@@ -275,7 +275,7 @@ VF.Test.GraceNote = (function () {
       voice.addTickables(createNoteBlock(['d/4', 'a/4'], 1));
       voice.addTickables(createNoteBlock(['d/4', 'a/4'], -1));
 
-      new vf.Formatter().joinVoices([voice]).formatToStave([voice], stave);
+      vf.Formatter().joinVoices([voice]).formatToStave([voice], stave);
```

Comment:
I think that this one requires new. I have included this call in the migrated music.ts and I get the following

![image](https://user-images.githubusercontent.com/22865285/112215674-eb552500-8c20-11eb-8cfa-cdbc44a3d3ce.png)

On this test I have also realised that the line 274 above also requires new.

![image](https://user-images.githubusercontent.com/22865285/112216146-76361f80-8c21-11eb-9cd4-cd170ca585d7.png)

Reply:
yeah, shouldn't running all tests tell you all the errors? (at least if there are none)

Reply:
Guys, please read the comment to this PR. vf is an object of class Factory, which has Formatter method. So it does not require new

Reply:
```typescript
var vf = VF.Test.makeFactory(options, 300);
vf.Formatter().joinVoices([voice, voice2]).formatToStave([voice, voice2], stave); 
```

Reply:
```typescript
export class Factory {
  Formatter(options) {
    return new Formatter(options);
  }
}
```

Reply:
you are right @wassertim it is a factory class, please ignore my comment.